pkg/acl/fuzz_test.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Because fuzz tests can be compute-intensive, should we add a build tag so they're only run when those tests are invoked?

for example, build tag `fuzztesting` and then we add a command to the makefile `make test-fuzz`?